gearmand/tox.ini

```diff
@@ -19,3 +19,11 @@ passenv =
 commands =
     pip install -r requirements.in
     pytest -v {posargs}
+setenv =
+    DOCKER_IMAGE=artefactual/gearmand
+    DOCKER_TAG=1.1.18-alpine
+    GEARMAND_VERSION=1.1.18
+    1.0.6: DOCKER_IMAGE=kendu/gearman
+    1.0.6: DOCKER_TAG=latest
+    1.0.6: GEARMAND_VERSION=1.0.6
```

Comment:
This might break if the latest version is updated. I think it's better to skip for latest like this:

https://github.com/DataDog/integrations-core/blob/1bf1bcf5f4af4a54f16bf45ad76d55c4f364ce05/clickhouse/tests/test_clickhouse.py#L58-L59

Reply:
This is an exception, and one of the reasons for the PR actually.

The image is four years old, and only has a single `latest` tag on it: https://hub.docker.com/r/kendu/gearman/tags